### DISCUSSION TITLE
Custom download path

### DISCUSSION THREAD

**Discussion by KRWCLASSIC** [bromite/bromite#2021] [Ideas]
third button when choosing where to download file that allows you to type your own download path (cuz you have to select between android/.../com.bromite.bromite/download folder in sdcard or download folder in main drive) so that would be cool new feature

**Comment by VA1DER** [comment]
One of the stated features is being able to select the download location. I can see nowhere to do this - there seems to be only two choices, either Bromite's selected internal folder, or Bromite's selected external folder.  Is there any way to customize those locations?
